src/rocm_docs/theme.py

```diff
@@ -24,19 +36,38 @@ def _update_repo_opts(srcdir: str, theme_opts: Dict[str, Any]) -> None:
         theme_opts.setdefault(key, val)
 
 
+def _set_theme_announcement(
+    announcement: str, theme_opts: Dict[str, Any]
+) -> None:
+    theme_opts.setdefault("announcement", announcement)
+
+
+def _update_banner(url: str, branch: str, theme_opts: Dict[str, Any]) -> None:
+    mapping_file_loc = "data/theme_announcement.yaml"
+    announcement_yaml = (
+        importlib_resources.files("rocm_docs") / mapping_file_loc
+    )
+
+    with open(announcement_yaml) as file:
+        banner_dict = yaml.safe_load(file)
```

Comment:
I don't think we need an extra file for this, let's just have the announcement either here in the code or in the html template.

Reply:
[4f409e0](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/261/commits/4f409e071eebc8d15d15eb8dedbee7aaf600400b) removes this yaml